doc/performance-testing/cloud-run/0003_stress-test-january-2022.md

```diff
@@ -0,0 +1,90 @@
+| Phase 3 | 115 | 50 | 20 mins | 100  | 4668 |  61.5 | 167 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-28T10:26:34)|
+
+## Results
+
+| Load Injector Instances | Requests per second | 99th percentile Max CPU Usage (%) | 99th percentile response time (ms) | Error rate (%) | Output |
+| --- | --- | --- | --- | --- | --- |
+| 45  | 5240 | 70.5 | 166 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-28T11:12:11) |
+| 46  | 5369 | 71.5 | 156 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-28T11:53:32) |
+| 47  | 5307 | 91.5 | 264 | 0.010 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-28T12:53:29) |
+| 48  | 5579 | 82.5 | 170 | 0.001 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-28T13:39:34) |
+| 49  | 5669 | 83.5 | 316 | 0.003 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-28T14:24:01) |
+| 50  | 5772 | 81.5 | 401 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-31T11:34:33) |
+| 51  | 5868 | 83.5 | 500 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-31T12:14:21) |
+| 52  | 5828 | 82.5 | 884 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-31T12:54:39) |
+| 53  | 6112 | 82.5 | 331 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-31T13:25:53) |
+| 54  | 6223 | 83.5 | 322 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-31T14:05:54) |
+| 55  | 6376 | 83.5 | 178 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-31T14:49:48) |
+| 56  | 6204 | 93.5 | 1173 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-02-01T08:13:20)|
+| 57  | 5960 | 90.9 | 2022 | 4.400 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-02-01T08:49:47)|
+| 58  | 6211 | 85.5 | 1508 | 5.650 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-02-01T09:24:10)|
+| 59  | 2953 | 91.5 | 1527 | 8.400 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-02-01T10:40:03)|
+
+## Errors and Failures
```

Comment:
The `Expected a (302) but got a (0) back when posting page` are Locust errors aren't they?

For all errors there should be underlying error cause text in the load balancer logs. Those are more relevant as this is what we'd be acting on.

Equally, the 502 errors should have an underlying